src/components/Blinds/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from "react";
+import blindsOpen from "../Blinds/blinds-open.svg";
+import blindsClosed from "../Blinds/blinds-closed.svg";
+import "../Blinds/style.css";
+
+const Blinds = ({ state }) => {
+  const [blinds, setBlinds] = useState({ state });
```

Comment:
Jelikoz my chceme aby se do blinds statu dostal retezec 'open' abychom pak s nim dale mohly pracovat, tady by stacilo zmenit `useState({state})` na `useState(state)`. Tak jak to mas ted napsane, neni dobre, protoze nastavujes object do statu misto retezce. Kvuli tomu ti nefunguje ani defaultni hodnota zaluzii. Kdyz otevru appku, nesviti ani `Otevreno` ani `Zavreno`.